es/lessons/basics/documentation.md

```diff
@@ -0,0 +1,309 @@
+---
+version: 1.0.1
+title: Documentación
+redirect_from:
+  - /lessons/basics/documentation/
+---
+
+Documentando código Elixir.
+
+{% include toc.html %}
+
+## Anotación
+
+Cuánto documentamos y qué hace que la documentación de calidad siga siendo un tema polémico dentro del mundo de la programación. Sin embargo, todos podemos estar deacuerdo que la documentación es importante para nosotros y para aquellos que trabajan con nuestro código.
```

Comment:
I would put it this way:
Cuánto y qué documentamos, hace que la documentación de calidad siga siendo un tema polémico dentro del mundo de la programación. Sin embargo, todos podemos estar de acuerdo en que la documentación es importante para nosotros y para aquellos que trabajan con nuestro código.